tests/shared_data.py

```diff
@@ -18,15 +18,48 @@
 
 TEST_TOKEN_SECRET = "secret"
 TEST_TOKEN_AUD = "account"
+TEST_TOKEN_SIGNING_ALG = "HS512"
 
 ISS = "https://bentov2auth.local/realms/bentov2"
 CLIENT = "local_bentov2"
 SUB = "david"
 
+TEST_IDP_SUPPORTED_TOKEN_SIGNING_ALGOS = [  # only HS* for testing purposes
+    "HS256",
+    "HS384",
+    TEST_TOKEN_SIGNING_ALG,
+]
+
+TEST_DISABLED_TOKEN_SIGNING_ALGOS = [  # disable all supported algos except the strongest one
+    alg for alg in TEST_IDP_SUPPORTED_TOKEN_SIGNING_ALGOS if alg != TEST_TOKEN_SIGNING_ALG
+]
+
 TEST_TOKEN = {
     "iss": ISS,
     "sub": SUB,
     "aud": TEST_TOKEN_SECRET,
+    "alg": TEST_TOKEN_SIGNING_ALG,
```

Comment:
alg is not a JWT claim, it is in the header - so this shouldn't be included here / in token bodies.